package/webhook/manifests.yaml

```diff
@@ -5,6 +5,25 @@ metadata:
   creationTimestamp: null
   name: validating-webhook-configuration
 webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validate-exoscale-crossplane-io-v1-bucket
+  failurePolicy: Fail
+  name: buckets.exoscale.crossplane.io
+  rules:
+  - apiGroups:
+    - exoscale.crossplane.io
+    apiVersions:
+    - v1
+    operations:
+    - UPDATE
```

Comment:
Don't forget to add CREATE if we check provider config in the webhook.

Reply:
Ah yes, I forgot about the provider config validation. I will add it.